content/apps/django.md

```diff
@@ -40,6 +38,8 @@ dj-database-url==0.3.0 #grabs environment variables and dumps them into a Django
 waitress==0.8.9 #a pure python WSGI server that is a replacement for gunicorn
 ```
 
+Note that [you will need to vendor the dependencies](http://docs.cloudfoundry.org/buildpacks/python/#vendoring) before deploying.
```

Comment:
Asked for clarification in https://github.com/cloudfoundry/docs-buildpacks/issues/83.